libs/search-widget/src/common/infinite-scroll/InfiniteScroll.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+  import { onMount, onDestroy, createEventDispatcher } from 'svelte';
+
+  export let hasMore = true;
+
+  const dispatch = createEventDispatcher();
+  let isLoadMore = false;
```

Comment:
did you meant `isLoadingMore` or `loadingMore`?

Reply:
`mustLoadMore` is more accurate